feature/notification/src/commonMain/kotlin/org/mifos/mobile/feature/notification/NotificationScreen.kt

```diff
@@ -105,8 +104,9 @@ private fun NotificationScreen(
                     is NotificationUiState.Success -> {
                         if (uiState.notifications.isEmpty()) {
                             EmptyDataView(
```

Comment:
create another state for Empty state. sth like this:
```
is -> NotificationUiState.Empty {
      EmptyDataView
}
is -> NotificationUiState.Success {

}
```
have a look at this PR how i did: https://github.com/openMF/mifos-mobile/pull/2774/files